### ISSUE TITLE
imported bin file doesn't work if not called immediately

### ISSUE THREAD

**Issue by denigada** [nwjs/nw.js#6709]
When I use it directly like this:
`<script>
        nw.Window.get().evalNWBin(null, 'enter.bin');
    </script>`

It works perfectly but when I use a jquery .load and import the binary in the callback, it doesn't work. Imported binary doesn't print anything in the console. I am loading my HTML content from the web, thats why I need to wait for it to load, to initialize the binary file, how can I fix this?

In short, this does not work:
`setTimeout(function() {
                nw.Window.get().evalNWBin(null, 'enter.bin');
        }, 2000);`

Also, getting the binary from a remote server stopped working for me. I think it is the same issue as above. It doesn't work when I initialize the binary file in `xhr.onload `

**Comment by denigada** [comment]
now it doesn't even work either way. I am having strange problems here. Sometimes it works after I create the binary file, then it stops working after testing 5-6 times

**Comment by denigada** [comment]
My app almost completely stopped working. I cannot use binary to run the app anymore. Please somebody help me

**Comment by Christywl** [comment]
Do you open Devtools on startup(before running `setTimeout(function() { nw.Window.get().evalNWBin(null, 'enter.bin'); }, 2000);`)?

**Comment by denigada** [comment]
@Christywl thanks for saving my life
